### ISSUE TITLE
GameGuru MAX - Bug - Behavior Library Not Alphabetized

### ISSUE THREAD

**Issue by MonkeyFrogStudio** [TheGameCreators/GameGuruRepo#4743]
As the title says, when you have the Behavior Library sorted alphabetically, the Behaviors are not in alphabetical order.

![bl](https://github.com/TheGameCreators/GameGuruRepo/assets/36735507/1572e9ba-3d8c-49c6-bf74-d2df39142efe)

In the screen shot above, you can see we go from:

- Boat
- Door
- Fuel
- Fuse
- Grow
- Spin
- Hover
- Mines
- Money
- Valve
- Add Fx
- Etc

It makes it a bit more difficult to find the Behavior you're looking for (of course, you can use search).

**Comment by davetgc** [comment]
A good point, but as you say, you can use search.

**Comment by MonkeyFrogStudio** [comment]
@davetgc This is a BUG, not an enhancement. If you look at the Behavior Library (and object library) at the upper-right corner, you  have a drop-down offering several ways to SORT them. The one I have circled in the screen shot is Name A-Z. This means that the Behavior Library should be sorted by name from A to Z. But, as shown, it doesn't do this. If I choose Name Z - A, then it doesn't do this, either.

In other words, these options are given, BUT THEY DON'T WORK. So, this is not an enhancement, but a BUG.

**Comment by LeeBamberTGC** [comment]
@MonkeyFrogStudio I can confirm it also happens with Object viewing too - on it :)

**Comment by MonkeyFrogStudio** [comment]
@LeeBamberTGC Yeah, I had reported the Object Library here:

https://github.com/TheGameCreators/GameGuruRepo/issues/3150

But had forgotten until after I'd reported the Behavior Library yesterday.

**Comment by LeeBamberTGC** [comment]
@MonkeyFrogStudio Fixed in next EXP build

**Comment by MonkeyFrogStudio** [comment]
@LeeBamberTGC This appears to be working on my end! Thanks for the quick fix!!!
